File-Formats/DocIO/Create-Word-document-in-MAUI.md

```diff
@@ -25,7 +25,7 @@ Step 2: Enter the project name and click **Create**.
 
 Step 3: Install the Syncfusion.DocIO.NET NuGet package as a reference to your project from [NuGet.org](https://www.nuget.org/).
 
-![Install the DocIO .NET NuGet package](MAUI_Images/Install_Nuget.png)
+![Install the DocIO .NET NuGet package in File Formats DOCIO](maui_images/file-formats-docio-install-nuget.png)
```

Comment:
1. Previous alt text seems fine. Revert Your alt changes in all places.
2. Change image name like as below,
Install-nuget-in-maui-word

here MAUI is the platform name, word is the control name. DOCIO means word. Update for all places.

Reply:
Reverted alt changes in all places and corrected image names.